### PR TITLE
Changelog v1.13.2

### PR DESCRIPTION
**Pull request by 6543**
as title ... need some fine tuning ...

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,37 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.13.2](https://github.com/go-gitea/gitea/releases/tag/v1.13.2) - 2021-01-31
+
+* SECURITY
+  * Add secure/httpOnly attributes to the lang cookie (#14279) (#14280)
+* API
+  * Check release publisher exists (#14375)
```
Comment (6543) [inline CHANGELOG.md]: "If release publisher is deleted use ghost user" ?

**Comment by root360-StefanHeitmueller** [comment]
Looks like the backport label is missing for #14530

**Review by khmarbaise — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (6543) [inline CHANGELOG.md]:
```suggestion
  * If release publisher is deleted use ghost user (#14375)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,37 @@ This changelog goes through all the changes that have been made in each release
...
+* BUGFIXES
+  * Set the name Mapper in migrations (#14526) (#14529)
+  * Fix wiki preview (#14515)
+  * Update code.gitea.io/sdk/gitea v0.13.1 -> v0.13.2 (#14497)
+  * ChangeUserName: rename user files back on DB issue (#14447)
+  * Fix lfs preview bug (#14428) (#14433)
+  * Ensure timeout error is shown on u2f timeout (#14417) (#14431)
+  * Fix Deadlock & Delete affected reactions on comment deletion (#14392) (#14425)
+  * Prevent panic on fuzzer provided string (#14405) (#14409)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,37 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.13.2](https://github.com/go-gitea/gitea/releases/tag/v1.13.2) - 2021-01-31
+
+* SECURITY
+  * Add secure/httpOnly attributes to the lang cookie (#14279) (#14280)
```
Comment (6543) [inline CHANGELOG.md:11]:
```suggestion
  * Prevent panic on fuzzer provided string (#14405) (#14409)
  * Add secure/httpOnly attributes to the lang cookie (#14279) (#14280)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,37 @@ This changelog goes through all the changes that have been made in each release
...
+* API
+  * Check release publisher exists (#14375)
+* BUGFIXES
+  * Set the name Mapper in migrations (#14526) (#14529)
```
Comment (6543) [inline CHANGELOG.md:16]:
```suggestion
  * Internal ssh server respect Ciphers, MACs and KeyExchanges settings (#14523)
  * Set the name Mapper in migrations (#14526) (#14529)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,38 @@ This changelog goes through all the changes that have been made in each release
...
 
+## [1.13.2](https://github.com/go-gitea/gitea/releases/tag/v1.13.2) - 2021-01-31
+
+* SECURITY
+  * Prevent panic on fuzzer provided string (#14405) (#14409)
+  * Add secure/httpOnly attributes to the lang cookie (#14279) (#14280)
+* API
+  * If release publisher is deleted use ghost user (#14375)
+* BUGFIXES
+  * Internal ssh server respect Ciphers, MACs and KeyExchanges settings (#14523)
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
  * Internal ssh server respect Ciphers, MACs and KeyExchanges settings (#14523) (#14530)
```

**Review by lafriks — APPROVED** [review]

**Pull request merged**